docs/docs/sdk/prediction-import/kitti.mdx

```diff
@@ -83,46 +86,57 @@ The JSON class map needs to follow the following structure:
 }
 ```
 
-The keys should correspond to the `featureNodeHash` of a bounding box object in the project ontology.
-To list the available hashes from your project, you can do this in your script:
+:::info
+
+To obtain a list of available `featureNodeHash`es for your project, you can use the following script:
 
 ```python
 from pathlib import Path
 
-import yaml
+from encord_active.lib.project.metadata import fetch_project_meta
 from encord import EncordUserClient
 
-data_dir = Path("/path/to/the/data")
-meta = yaml.safe_load((data_dir / "project_meta.yaml").read_text())
-private_key = Path(meta["ssh_key_path"]).read_text()
+project_path = Path("/path/to/your/project/folder")
+project_meta = fetch_project_meta(project_path)
+private_key = Path(project_meta["ssh_key_path"]).read_text()
 
 client = EncordUserClient.create_with_ssh_private_key(private_key)
-project = client.get_project(project_hash=meta["project_hash"])
+encord_project = client.get_project(project_hash=project_meta["project_hash"])
 
-# NB: Remember to include the first code snippet on this page.
-print({o["featureNodeHash"]: o["name"] for o in project.ontology["objects"]})
-# Outputs somtihing similar to
+print({o["featureNodeHash"]: o["name"] for o in encord_project.ontology["objects"]})
+# Example output:
```

Comment:
Since we're updating this anyway - it would probably be better to just load the local `ontology.json` file instead and use that. That would remove the requirement of the project being uploaded to Annotate.

Reply:
purfect observation 👍